index.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style/style.css" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <title>YouTube</title>
+    </head>
+    <body>
+        <!--Start of navigation bar-->
+        <header>
+            <nav>
+                <a href="#"><img class="bars" class="img" src="img/img_bar.png" alt="image bar"></a>
+                <a href="#"><img class="logo" src="img/youtube_logo.png" alt="YouTube"></a>
+                <input class="search" type="text" placeholder="Search" class="img">
+                <a href="#"><button id="loop"><i class="fa fa-search" aria-hidden="true"></i></button></a>
+                <a href="#"><img class="upload" class="img" src="img/img_arrow.png" alt="image arrow"></a>
```

Comment:
- [ ] The sizing of this issue has been messed up. It is currently way too large. Please resize it. 
![Screen Shot 2020-07-14 at 1 00 35 AM](https://user-images.githubusercontent.com/48804258/87361864-6f70ec00-c56d-11ea-873d-9c13b42623ed.png)